worker/uniter/actions/resolver_test.go

```diff
@@ -190,3 +240,12 @@ type mockFailOp struct {
 func (op *mockFailOp) String() string {
 	return op.name
 }
+
+type mockSkipHookOp struct {
```

Comment:
I'm curious why we need to create a new skip hook mock when we already have code which creates a skip hook op. I wonder how that was tested.
```
	case operation.RunAction:
		if localState.Hook != nil {
			r.logger.Infof("found incomplete action %v; ignoring", localState.ActionId)
			r.logger.Infof("recommitting prior %q hook", localState.Hook.Kind)
			return opFactory.NewSkipHook(*localState.Hook)
		}
```

Reply:
I guess it wasn't tested.

Reply:
Added a test for the existing behaviour